src/Component/Modal/ApplicationInfo/ApplicationInfo.tsx

```diff
@@ -43,6 +47,10 @@ export const ApplicationInfo: React.FC<ApplicationInfoProps> = ({
         className="shogun-logo"
         src={defaultLogo}
       />
+      <Statistic
+        title="Admin Version"
```

Comment:
Very minor, but it would be very helpful if this value would be added to the i18n.